src/email.js

```diff
@@ -0,0 +1,41 @@
+const SMTPServer = require("smtp-server").SMTPServer;
+const MailParser = require('mailparser').MailParser;
+const addrs = require("email-addresses");
+
+const ROOM_ID = "!IfhHtxETxbbvbBgfol:localhost"; // TODO: Join rooms based on alias and get the roomID
+
+
+module.exports.startSMTP = new SMTPServer({
+    secure: false,
+    logger: false,
+    disabledCommands: ['AUTH', 'STARTTLS'],
+    authOptional: true,
+
+    onData: function (stream, session, callback) {
+        let subject, text;
+        let mailparser = new MailParser();
+        let fromAdd = addrs.parseOneAddress(session.envelope.mailFrom.address);
```

Comment:
I think these can be const?

```suggestion
        const mailparser = new MailParser();
        const fromAdd = addrs.parseOneAddress(session.envelope.mailFrom.address);
```